contributors/devel/sig-architecture/api-conventions.md

```diff
@@ -5,7 +5,7 @@ Updated: 3/7/2017
 
 *This document is oriented at users who want a deeper understanding of the
 Kubernetes API structure, and developers wanting to extend the Kubernetes API.
-An introduction to using resources with kubectl can be found in [the object management overview](https://kubernetes.io/docs/tutorials/object-management-kubectl/object-management/).*
+An introduction to using resources with kubectl can be found in [the object management overview](https://kubernetes.io/docs/tasks/manage-kubernetes-objects/declarative-config/).*
```

Comment:
nit: can we replace "object management overview" with "declarative object management overview" or something like that?

A page for imperative object management exists as well (https://kubernetes.io/docs/tasks/manage-kubernetes-objects/imperative-config/), so I think it'd be helpful to be precise here. :)

Reply:
Done